Formula/aerc.rb

```diff
@@ -16,9 +17,12 @@ class Aerc < Formula
   end
 
   depends_on "go" => :build
+  depends_on "make" => :build
   depends_on "scdoc" => :build
 
   def install
+    ENV.prepend_path "PATH", Formula["make"].opt_libexec/"gnubin"
+
```

Comment:
This evades our compiler shims.
```suggestion
```
Instead, replace the calls to `make` with `gmake` below.

Reply:
When I tried that, it didn't work. I did some prints and it was getting `gmake` from `/opt/homebrew/Library/Homebrew/shims/mac/super/gmake`, which is a shell script to execute system make.